Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -335,10 +335,9 @@ extension ComponentAPICoordinator {
                 case .success(let extractions):
                     self.handleAnalysis(with: extractions)
                 case .failure(let error):
-                    let error = error as? AnalysisError ?? AnalysisError.unknown
-                    guard error != .cancelled else { return }
+                    guard error != .requestCancelled else { return }
 
-                    self.showErrorInAnalysisScreen(with: error.message) {
+                    self.showErrorInAnalysisScreen(with: AnalysisError.unknown.message) {
```

Comment:
So you disregard the `error` object and always show an "unknown" error? Are you sure that's fine? Can you at least log it?

Reply:
Before doing this I checked the trace of the error and it was always showing this error, so didn't want to make it more complicated. 
It's already logged in the networking side